src/PointwiseFunctions/Hydro/MassWeightedFluidItems.hpp

```diff
@@ -60,6 +60,15 @@ struct MassWeightedCoords : db::SimpleTag {
 };
 }  // namespace Tags
 
+
+template <typename DataType, size_t Dim, typename Frame>
+void u_lower_t(const gsl::not_null<Scalar<DataType>*> result,
```

Comment:
Add documentation?

Reply:
Hi where should I add the documentation?

Reply:
Just a few lines up, line 63.
```cpp
/// \brief DOCUMENTATION HERE
template <typename DataType, size_t Dim, typename Frame>
void u_lower_t(const gsl::not_null<Scalar<DataType>*> result,
```